src/ankisyncd/sync_app.py

```diff
@@ -113,7 +113,7 @@ def usnLim(self):
     # server-side deletions to be returned by start
     def start(self, minUsn, lnewer, graves={"cards": [], "notes": [], "decks": []}, offset=None):
         if offset is not None:
```

Comment:
This if-statement is redundant if we're no longer raising an exception. It should either be removed or a log message should be included.